index.html

```diff
@@ -1317,6 +1361,34 @@
     <a>Web NFC message</a>s either from an <a>NFC tag</a> or an <a>NFC peer</a>.
   </p>
 
+  <section> <h3>The <strong>NFCPushTarget</strong> enum</h3>
+    <dl title="enum NFCPushTarget" class="idl">
+      <dt>tag</dt>
+      <dt>peer</dt>
+      <dt>any</dt>
+    </dl>
+  </section>
+
+  <section> <h3>The <strong>NFCPushOptions</strong> dictionary</h3>
+    <dl title="dictionary NFCPushOptions" class="idl">
```

Comment:
Consider switching over to [contiguous IDL](http://www.w3.org/respec/examples/webidl-contiguous.html#introduction). It supports more of IDL and makes it easier to link things (no more `widl-long-string`).

Eventually, you might consider switching to [Bikeshed](https://github.com/tabatkins/bikeshed), which does a better job of cross-linking between specifications, but that's a fairly large investment.